core-plugin/src/com/google/gct/idea/debugger/CloudDebugProcess.java

```diff
@@ -501,8 +501,17 @@ public SaveAndExitAction() {
 
     @Override
     public void actionPerformed(AnActionEvent event) {
-      ExitDialog exitDialog = new ExitDialog(getXDebugSession().getProject());
```

Comment:
OK. Change the button names and if we're not using it delete that class and its tests.